src/main/java/seedu/address/model/person/UniqueLoanList.java

```diff
@@ -353,7 +354,8 @@ public Date getEarliestReturnDate() {
         Date earliestReturnDate = null;
         for (Loan loan : internalList) {
             if ((earliestReturnDate == null || loan.getReturnDate().before(earliestReturnDate))
-                    && !loan.getReturnDate().before(new Date()) && !loan.isReturned()) {
+                    && !loan.isOverdue() && !loan.isReturned()) {
+                System.out.println(loan.getReturnDate() + " " + loan.isOverdue() + " " + loan.isReturned());
```

Comment:
Would you want to remove this `System.out.println` statement? HAHA

Reply:
Oops, missed it. I'll remove the sysout lines